src/operations/find_and_modify.ts

```diff
@@ -205,7 +213,7 @@ class FindAndModifyOperation extends CommandOperation<Document> {
     // Execute the command
     super.executeCommand(server, session, cmd, (err, result) => {
       if (err) return callback(err);
-      return callback(undefined, result);
+      return callback(undefined, options.includeResultMetadata ? result : result.value ?? null);
```

Comment:
When is `value` nullish and has it been null or undefined? 

I might be missing them but I'm not seeing a test for the `null` case

Reply:
Added tests for the non-matching null cases for all 3 methods now.